src/OpenTelemetry.AutoInstrumentation/Logging/InternalLogger.cs

```diff
@@ -214,7 +214,7 @@ private void WriteImpl(LogLevel level, Exception? exception, string messageTempl
     {
         try
         {
-            var rawMessage = string.Format(messageTemplate, args);
+            var rawMessage = args.Length > 0 ? string.Format(messageTemplate, args) : messageTemplate;
```

Comment:
Is `args` guaranteed to never be `null` here? If yes, can the nullable be removed?

Anyway, instead of changing the implementation here wouldn't be better to follow the patterns as the prior code?

What do you think @lachmatt ?

Reply:
Correct. Misclassified this as a bug, instead double formatting causes the bug.

Reply:
Reverted.